design-proposals/enhancements.md

```diff
@@ -0,0 +1,109 @@
+4. to make one SIG to own the feature once it has been merged. The SIG is responsible for maintaining, fixing, running, _everything-it_.
+
+This has a few effects - see the following Goals section.
+
+## Goals
+
+1. The design process now has a mechanism to distribute designs among SIGs
+2. SIG approvers are empowered to approve designs, increase the approvers pool
+3. The ownership of an implemented feature becomes clear
+4. Ensure that designs converge (accept, reject)
+5. Formalize this process as an Virtualization Enhancement Proposal (VEP) process
+
+## Non Goals
+
+1. Create unnecessary paperwork
+
+## Definition of Users
+
+* VEP Author - The person writing a design/enhancement proposal
+* SIG - A formal KubeVirt SIG
+* SIG Approver - A member of a SIG with approval permissions
+
+## User Stories
+
+* As a VEP Author I want to know who I can work with in order to move
+  my proposal forward
+* As a SIG I want to have a say in what is getting pushed into my domain
+  in order to make sure that we are able to maintain it
+* As a SIG Approver I want to ensure that a design is sound before a
+  VEP Author is approaching an implementation.
+* As a KubeVirt developer, I want to have a single source-of-truth
+  w.r.t. the current status, design API of a feature.
+
+## Repos
+
+- https://github.com/kubevirt/enhancements
+
+# Design
+
+Key elements:
+
+- Ownership: SIGs own a _feature_ (which includes the process, but also
+  the resulting code) from it's inception (design) all the way (fixes, maintenance) to it's end (removal)[^1]
+- Approvals: SIG approvers will be allowed to approve designs
+- Responsibilities: SIG Approvers are responsible for driving a design, and connecting it to other SIGs as needed
+
+Process elements:
+
+- VEP Author creates a GitHub Issue for getting a unique identifier and starting the process
+- VEP Author creates a PR to propose the design targeting a specific SIG
+- SIG decides on an approver to shepherd the VEP
+- SIG collaborates with other SIGs to ensure its thoroughly reviewed
+- SIG approves or rejects VEP
+- Other SIG approvers can veto a proposal
+- SIG owns future maintenance of the implementation
+- KubeVirt Maintainers are responsible to support the owning SIG and VEP author in the case of conflicts and questions
+
+Technical elements:
+
+- VEPs will live in a new dedicated repository `kubevirt/enhancements`
+- `OWNER_ALIASES` will be mirrored from kubevirt/kubevirt in order to have the same SIGs in the EP repository
+- Approvals and ownership is defined with `OWNERS` files in the `veps/sig-*` directories, tying into the general prow approval and merge flow
+- GitHub Issues will be used to create unique identifiers
+
+## API Examples
+
+None.
+
+## Scalability
+
+Instead of relying on a small approvers pool, now the process starts
+with routing VEPs in the beginning of their life-time to SIGs.
+This is expected to increase the time-to-merge-or-reject for VEPs.
```

Comment:
Does this really increase the time? I would think spreading the load across a larger pool of possible approvers should speed up the process?